tools/importer-rest-api-specs/components/dataapigeneratorjson/templater_terraform_resource_mappings.go

```diff
@@ -3,40 +3,16 @@ package dataapigeneratorjson
 import (
 	"encoding/json"
 
+	"github.com/hashicorp/pandora/tools/importer-rest-api-specs/components/dataapigeneratorjson/models"
```

Comment:
for consistency (and since there's already a `models` package within `importer-rest-api-specs`) - could we import-alias this for now?

```suggestion
	dataApiModels "github.com/hashicorp/pandora/tools/importer-rest-api-specs/components/dataapigeneratorjson/models"
```